src/shipit_workflow/shipit_workflow/release.py

```diff
@@ -55,3 +55,19 @@ def bump_version(version):
         v.append('0')
     v[-1] = str(int(v[-1]) + 1)
     return split_by.join(v) + suffix
+
+
+def get_beta_num(version):
+    if is_beta(version):
+        parts = version.split('b')
+        return int(parts[-1])
+
+
+def is_partner_enabled(product, version):
+    if product == 'firefox':
+        if is_beta(version):
+            if get_beta_num(version) >= 8:
```

Comment:
We probably want to config this at some point - we may want partner repacks all the time in staging, f/e.